jupyter-pytorch-full/tox.ini

```
@@ -12,29 +13,46 @@ setenv =
     CHARM_BRANCH=main
     LOCAL_CHARM_DIR=charm_repo
 
-[testenv:unit]
+[testenv:pack]
 passenv = *
 allowlist_externals =
-    bash
-    tox
     rockcraft
-deps =
-    charmed-kubeflow-chisme
-    juju<4.0
-    pytest
-    pytest-operator
-    ops
-    tenacity
 commands =
-    # build and pack rock
     rockcraft pack
+
+[testenv:export-to-docker]
+passenv = *
+allowlist_externals =
+    bash
+    skopeo
+    yq
+commands =
+    # export already packed rock to docker
     bash -c 'NAME=$(yq eval .name rockcraft.yaml) && \
              VERSION=$(yq eval .version rockcraft.yaml) && \
-             ARCH=$(yq eval -r ".platforms | keys" rockcraft.yaml | cut -d" " -f2) && \
-             ROCK="$\{NAME\}_$\{VERSION\}_$\{ARCH\}" && \
-             sudo skopeo --insecure-policy copy oci-archive:$ROCK.rock docker-daemon:$ROCK:$VERSION && \
-             docker save $ROCK > $ROCK.tar'
+             ARCH=$(yq eval ".platforms | keys | .[0]" rockcraft.yaml) && \
+             ROCK="$\{NAME\}_$\{VERSION\}_$\{ARCH\}.rock" && \
+             DOCKER_IMAGE=$NAME:$VERSION && \\
```

```suggestion
             DOCKER_IMAGE=$NAME:$VERSION && \
```
I know since I 'm the one who introduced that at first :smiley:

---

fixed